src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -220,7 +220,7 @@ private void testCreateInstructorAction() throws Exception {
         coursesPage.loadInstructorHomeTab();
         instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
         instructorHomePage.clickFeedbackSessionViewResultsLink(demoCourseId, "Second team feedback session")
-                          .waitForPageToLoad();
+                          .expandPanels();
```

Comment:
I'm for skipping the expanding parts as well, but it will cause a huge amount of HTML change. We can do that as an extension after this issue.